ChatApp/Presentation/View/ViewController/ViewController.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  ViewController.swift
+//  ChatApp
+//
+//  Created by Nika Gogichashvili on 18.04.23.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    
+    // MARK: Properties
+    private lazy var stackView = UIStackView(arrangedSubviews: [topMessageView, dividerView, bottomMessageView])
```

Comment:
if you set up stack view lazily anyways you can remove setUpStackView method and move it's contents into the lazy initialisation